keps/sig-release/3000-artifact-distribution/README.md

```diff
@@ -0,0 +1,135 @@
+# KEP 3000: Artifact Distribution Policy
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [SIG Release - Image Promotion](#sig-release---image-promotion)
+    - [Cloud Customer - Installing K8s via kubeadm](#cloud-customer---installing-k8s-via-kubeadm)
+  - [Notes/Constraints/Caveats](#notesconstraintscaveats)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Artifact Promotion](#artifact-promotion)
+    - [Policy](#policy)
+    - [Process](#process)
+  - [Artifact Distribution](#artifact-distribution)
+    - [Policy](#policy-1)
+    - [Process](#process-1)
+- [Alternatives / Background](#alternatives--background)
+  - [How much is this going to save us?](#how-much-is-this-going-to-save-us)
+- [Infrastructure Needed](#infrastructure-needed)
+- [Hack on this doc](#hack-on-this-doc)
+<!-- /toc -->
+
+## Summary
+
+The container images and release binaries produced by our community need a clear path to be hosted by multiple service/cloud providers.
+
+The global community should be routed to the appropriate mirror for their country or cloud provider to ensure cost effective worldwide access.
+
+This KEP should cover the policy and distribution mechanisms we will put in place to allow creating a globally distributed, multi-cloud and country solution.
+
+## Motivation
+
+Currently we push to a single provider, and distributing to the rest of community comes at great cost nearing $150k/month (mostly egress) in donations.
+
+Additionally, some of our community members are unable to access the official release artifacts due to country level firewalls that do not them connect to Google services.
+
+Ideally we can dramatically reduce cost and allow everyone in the world to download the artifacts released by our community.
+
+### Goals
+
+A policy and procedure for use by SIG Release to promote container images and release binaries to multiple registries and mirrors.
+
+A solution to allow redirection to appropriate mirrors to lower cost and allow access from any cloud or country globally.
+
+### Non-Goals
+
+Anything related to creation of artifacts, bom, digital signatures.
+
+## Proposal
+
+There are two intertwined concepts that are part of this proposal.
+
+First, the policy and procedures to promote/upload our artifacts to multiple providers. Our existing processes upload only to GCS buckets. Ideally we extend the existing software/promotion process to push directly to multiple providers. Alternatively we use a second process to synchronize artifacts from our existing production buckets to similar constructs at other providers.
+
+Additionally we require a registry and artifact url-redirection solution to the local cloud provider or country.
+
+### User Stories
+
+#### SIG Release - Image Promotion
```

Comment:
Do we need to ensure the compatibility with `cosign` ? (if there is a relation with [KEP 3031](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/3031-signing-release-artifacts))

Reply:
cosign is compatible with OCI registries, which seems to be a trivial requirement for container registries. :+1: